src/Query/FilterParser.ts

```diff
@@ -87,10 +88,18 @@ export function parseGrouper(line: string): Grouper | null {
     for (const creator of fieldCreators) {
         const field = creator();
         const fieldName = field.fieldNameSingular();
-        if (line === `group by ${fieldName}`) {
-            if (field.supportsGrouping()) {
+        if (field.supportsGrouping()) {
+            if (line === `group by ${fieldName}`) {
                 return field.createGrouper();
             }
+
+            // MultiTextField is written as a plural
+            // See also MultiTextField.createGrouper()
+            if (field instanceof MultiTextField) {
+                if (line === `group by ${fieldName}s`) {
+                    return field.createGrouper();
+                }
+            }
```

Comment:
I think are reinventing `MultiTextField. fieldNamePlural()` here and it would be better to call that.

Reply:
that was my first instinct too so I tried that, but now that I gave it another try, I noticed that `fieldNamePlural()` was `protected` and so it moved it to `public` now